Applications/Help/main.cpp

```diff
@@ -159,10 +165,21 @@ int main(int argc, char* argv[])
         model->update_section_node_on_toggle(index, open);
     };
 
+    auto open_external = [](auto& url) {
+        if (!Desktop::Launcher::open(url))
+            dbg() << "Failed to open '" << url << "'";
```

Comment:
We need to actually notify the user, not just log it. Perhaps show a message box?

Reply:
I left in the added message box nonetheless.